charts/tidb-operator/values.yaml

```diff
@@ -182,13 +182,22 @@ admissionWebhook:
   serviceAccount: tidb-admission-webhook
   rbac:
     create: true
+  hooksEnabled:
+    ## statefulsets hook would check requests for updating tidbcluster's statefulsets
+    ## If enabled it, the statefulsets of tidbcluseter would update in partition by tidbcluster's annotation
+    statefulSets: false
+    ## pods hook would check requests for creating and deleting tidbcluster's pods
+    ## if enabled it, the pods of tidbcluster would safely created or deleted by webhook instead of controller
+    pods: false
```

Comment:
because we will always deploy webhook eventually because many features must be implemented in webhook (defaulting & validating, etc). Maybe we can have only one gate `hooksEnabled`?

cc @aylei

Reply:
I am ok with these fine-grained control, but I think the default value is better to be `true`. The UX I expect is that when I enable webhook by setting `admission.create=true`, I get all the admissions out-of-box, and could disable some of them based on my customized requirement.

Reply:
updated, the `podsWebhook` is `true` in default while the `stsWebhook` is `false` as `stsWebhook` is a special feature.